src/org/loklak/data/DAO.java

```diff
@@ -978,6 +987,25 @@ public SearchLocalMessages(final String q, final Timeline.Order order_field, fin
             }
             this.aggregations = query.aggregations;
         }
+
+        public SearchLocalMessages (
+                final String q,
+                final Timeline.Order order_field,
+                final int timezoneOffset,
+                final int resultCount,
+                final int aggregationLimit,
+                final String... aggregationFields
+        ) {
+            this(
+                q,
+                order_field,
+                timezoneOffset,
+                resultCount,
+                aggregationLimit,
+                new ArrayList<>(),
+                aggregationFields
+            );
+        }
```

Comment:
@singhpratyush This is very long when accomodated in one line, this indentation is to improves readibility. 
Don't you think it is more readable now?